tests/unittest/llmapi/test_llm_args.py

```
@@ -1741,3 +1741,47 @@ def test_threshold_scale_factor_unaffected(self):
+
+        cfg = SkipSoftmaxAttentionConfig(threshold_scale_factor=0.001)
+        assert cfg.stat_log_path is None
+
+    def test_stat_log_path_with_cuda_graph_raises(self, tmp_path):
+        from tensorrt_llm.llmapi.llm_args import (CudaGraphConfig,
+                                                  SkipSoftmaxAttentionConfig,
+                                                  TorchLlmArgs)
+
+        cfg = SkipSoftmaxAttentionConfig(threshold_scale_factor=0.001,
+                                         stat_log_path=str(tmp_path /
+                                                            "stat.json"))
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)